code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -2182,3 +2182,76 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 /mob/living/carbon/human/proc/apply_offstation_roles(source)
 	SIGNAL_HANDLER
 	mind.offstation_role = TRUE
+
+/mob/living/carbon/human/proc/expert_chef_knowledge()
+	set name = "Expert Chef Knowledge"
+	set desc = "Examine your surroundings, see what you can cook."
+	set category = "Martial Arts"
+
+	if(!HAS_TRAIT(mind, TRAIT_EXPERT_CHEF))
+		return
+	if(has_status_effect(STATUS_EXPERT_CHEF))
+		to_chat(src, "<span class='warning'>All this thinking makes your head hurt, wait a bit longer.</span>")
+		return
+	apply_status_effect(STATUS_EXPERT_CHEF)
```

Comment:
This doesn't read right. Please rename the status effect to something like STATUS_USED_EXPERT_CHEF